mentoring/tests/integration/xml/mcq_with_html_choices.xml

```diff
@@ -0,0 +1,22 @@
+<vertical_demo>
+    <mentoring url_name="mcq_with_comments" display_name="MCQ With Resizable popups" weight="1" enforce_dependency="false">
+        <mcq name="mrq_1_1_7" question="What do you like in this MCQ?" correct_choices="gracefulness,elegance,beauty">
+            <choice value="elegance"><b>Its elegance</b></choice>
```

Comment:
While we're here, can we use the likes of `<strong>` and `<em>`? This is a nit, but it would be nice. We even use `<strong>` a few lines down from here. It'd be nice for consistency.